src/commands/app/init.js

```diff
@@ -28,9 +34,30 @@ class InitCommand extends BaseCommand {
 
     aioLogger.debug('creating new app with init command ', flags)
 
+    // default project name and services
     let projectName = path.basename(process.cwd())
     let services = 'AdobeTargetSDK,AdobeAnalyticsSDK,CampaignSDK,McDataServicesSdk,AudienceManagerCustomerSDK' // todo fetch those from console when no --import
 
+    if (!(flags.import || flags.yes)) {
+      const accessToken = await getToken(CLI)
+      const { env: imsEnv = 'prod' } = await context.getCli()
```

Comment:
will this work if the CLI config is undefined ? (first time users)

Reply:
oops, yes you are right it will fail here. I'll add a failing test and fix it.

Reply:
updated with a fix commit